plugin/listtoggle.vim

```diff
@@ -18,11 +18,6 @@
 "
 " See ':help listtoggle' for more information.
 
-if exists('g:loaded_listtoggle')
-  finish
-endif
-let g:loaded_listtoggle = 1
```

Comment:
Why is this being removed? It's there to prevent problems with the plugin being loaded twice.